10-scroll/setup/app.js

```diff
@@ -1,13 +1,83 @@
-// Element.getBoundingClientRect() method returns the size of an element and its position relative to the viewport.
-// pageYOffset is a read - only window property that returns the number of pixels the document has been scrolled vertically.
-// slice extracts a section of a string without modifying original string
-//offsetTop - A Number, representing the top position of the element, in pixels
+// Element.getBoundingClientRect() method returns the size of an 
+// element and its position relative to the viewport.
+// pageYOffset is a read - only window property that returns the number
+// of pixels the document has been scrolled vertically.
+// slice extracts a section of a string without modifying original 
+// string
+//offsetTop - A Number, representing the top position of the element, 
+// in pixels
 
 // ********** set date ************
+const date = document.getElementById('date')
+date.innerHTML = new Date().getFullYear()
 
 // ********** close links ************
+const navToggle = document.querySelector('.nav-toggle');
+const linksContainer = document.querySelector('.links-container');
+const links = document.querySelector('.links');
 
+navToggle.addEventListener('click', function(){
+    //linksContainer.classList.toggle("show-links");
+    const containerHeight = linksContainer.getBoundingClientRect().height;
+    const linksHeight = links.getBoundingClientRect().height;
+
+    if(containerHeight === 0) {
+        linksContainer.style.height = `${linksHeight}px`
+    }
+    else {
```

Comment:
following [eslint](https://eslint.org/docs/latest/rules/callback-return#ifelse-statements) this should be togheter like...

`} else {`